lib/cc/engine/gem_detector.rb

```diff
@@ -0,0 +1,17 @@
+require "bundler/lockfile_parser"
+
+class GemDetector
+  def initialize(lockfile)
+    @contents = File.read(lockfile)
```

Comment:
WDYT about doing the `File.exist?` check here (and setting an empty `@content` if not present)?

*Slightly* dirty implicitly triggering the bundler exception=>false behavior with the empty content, but it would allow `Rubcop#cops` to be a cleaner one-level if-else, which I think is worth it.